src/pages/FamiliarizationPage.svelte

```diff
@@ -1 +1,20 @@
-<h2>Familiarization page</h2>
+<script lang="ts">
+    import { _ } from 'svelte-i18n'
+    import AlphanumericInput from '../components/AlphanumericInput/AlphanumericInput.svelte'
+    import { swiper } from '../stores'
+
+    const CHUNK_LENGTH = 9
+    const SEQUENCE = 'T890SOLA6 RF6628GB8'
+</script>
+
+<p>{$_('familirization_page.here_an_example')}</p>
+
+<p>{$_('familirization_page.instructions')}</p>
+
+<p class="numeric-readable fs-5">{SEQUENCE}</p>
+
+<AlphanumericInput
+  value={SEQUENCE}
```

Comment:
Better use `.replace(' ', '')` here:

```suggestion
  value={SEQUENCE.replace(' ', '')}
```

Reply:
:+1: